### PR TITLE
add workflow check_query_identifiers and dummy script #339

### PR DESCRIPTION
**Pull request by catreedle**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

This PR adds a new workflow that would run [check_query_identifiers.py](https://github.com/catreedle/Scribe-Data/blob/workflow-identifier/src/scribe_data/check/check_query_identifiers.py)
It has been tested on a forked repository using a dummy script.

Any feedback or suggestions are greatly appreciated.

### Related issue

- #339

### CODE REVIEW

**Comment by andrewtavis** [comment]
Can we get the branch actions to look like this, @catreedle:

```
  push:
    branches: [main]
  pull_request:
    branches:
      - main
    types: [opened, reopened, synchronize]
```

That way the workflow reruns on future commits? :)

**Comment by catreedle** [comment]
> Can we get the branch actions to look like this, @catreedle:
> 
> ```
>   push:
>     branches: [main]
>   pull_request:
>     branches:
>       - main
>     types: [opened, reopened, synchronize]
> ```
> 
> That way the workflow reruns on future commits? :)

Updated it. Thank you! @andrewtavis

**Review by andrewtavis — APPROVED** [review]
Commented out the actions in the workflow so that we can bring this in now and eventually bring it into @DeleMike's PR, @catreedle :) Wonderful work! Thanks so much 😊

File: .github/workflows/check_query_identifiers.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      #     uv venv
+      #     uv pip install -r requirements.txt
+
+      # - name: Activate virtualenv
+      #   run: |
+      #     . .venv/bin/activate
+      #     echo PATH=$PATH >> $GITHUB_ENV
+
+      # - name: Run Python script
+      #   run: python src/scribe_data/check/check_query_identifiers.py
```
Comment (DeleMike) [inline .github/workflows/check_query_identifiers.yaml:45]: Hello @catreedle and @andrewtavis, do you know why this was commented out? `check_query_identifiers.py` is done and I need to be sure all the commands are useful.

Comment (DeleMike) [reply .github/workflows/check_query_identifiers.yaml:45]: I wanna know why, in the steps, we are setting up Python, installing dependencies, and activating a virtual environment but `.github/workflows/check_project_structure.yaml` does not have this style? is there a reason why?

Comment (andrewtavis) [reply .github/workflows/check_query_identifiers.yaml:45]: This is commented out so we don't get errors before everything is hooked up :)

Comment (andrewtavis) [reply .github/workflows/check_query_identifiers.yaml:45]: Maybe we don't need the environment for this one and it'd make sense to simplify the steps 😊

Comment (DeleMike) [reply .github/workflows/check_query_identifiers.yaml:45]: Thanks @andrewtavis for the response!
yeah, maybe we don't need it. I can raise a PR and we quickly evaluate it :)

Comment (andrewtavis) [reply .github/workflows/check_query_identifiers.yaml:45]: Thanks, @DeleMike!

**Pull request merged**